src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxTree.DebuggerSyntaxTree.vb

```diff
@@ -0,0 +1,32 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.Text
+
+Namespace Microsoft.CodeAnalysis.VisualBasic
+
+    Partial Public MustInherit Class VisualBasicSyntaxTree
+        Private NotInheritable Class DebuggerSyntaxTree
```

Comment:
What makes this a debugger one versus a non-debugger one? Comment?

Reply:
It's used in EE. Mirrors C#.  Will comment.